tests/jballerina-unit-test/src/test/resources/test-src/statements/ifelse/type-guard.bal

```diff
@@ -367,7 +367,8 @@ type Person record {
 };
 
 type Student record {
-    *Person;
+    string name;
+    int age;
```

Comment:
There is a L1 identified with this - https://github.com/ballerina-platform/ballerina-lang/issues/15780